### ISSUE TITLE
Cannot find dependency , angular cli

### ISSUE THREAD

**Issue by MochaTheCoder** [highcharts/highcharts-angular#12]
Steps:

1) npm install highcharts --save
2) npm install highcharts/highcharts-angular
3)
```
 import { HighchartsChartComponent } from 'highcharts-angular';

@NgModule({
  declarations: [
    HighchartsChartComponent,
    ...]
```

4) ng -serve
```
WARNING in ./node_modules/highcharts-angular/node_modules/@angular/core/esm5/core.js
6561:15-102 Critical dependency: the request of a dependency is an expression
    at ImportLazyContextDependency.getWarnings (C:\Users\xxx\node_modules\webpack\lib\dependencies\ImportContextDependency.js:28:4)
    at Compilation.reportDependencyErrorsAndWarnings (C:\Users\xxx\node_modules\webpack\lib\Compilation.js:677:24)
    at Compilation.finish (C:\Users\xxx\node_modules\webpack\lib\Compilation.js:535:9)
    at applyPluginsParallel.err (C:\Users\xxx\node_modules\webpack\lib\Compiler.js:512:17)
    at C:\Users\xxx\node_modules\tapable\lib\Tapable.js:289:11
    at _addModuleChain (C:\Users\xxx\node_modules\webpack\lib\Compilation.js:481:11)
    at processModuleDependencies.err (C:\Users\xxx\node_modules\webpack\lib\Compilation.js:452:13)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

npm version : 5.6.0
ng - v :
```

 @angular/cli: 1.3.1
node: 6.10.2
os: win32 x64
@angular/animations: 4.3.6
@angular/common: 4.3.6
@angular/compiler: 4.3.6
@angular/core: 4.3.6
@angular/forms: 4.3.6
@angular/http: 4.3.6
@angular/material: error
@angular/platform-browser: 4.3.6
@angular/platform-browser-dynamic: 4.3.6
@angular/router: 4.3.6
@angular/cli: 1.3.1
@angular/compiler-cli: 4.3.6
@angular/language-service: 4.3.6
```

**Comment by KacperMadej** [comment]
Hi,

Thank you for reporting.

This looks like related to this issue https://github.com/webpack/webpack/issues/196

If update to Angular v5 the only change that is required seems to be this - https://github.com/highcharts/highcharts-angular#addition-for-angular-v5-to-resolve-a-build-fail

**Comment by gregjacobs** [comment]
Hey @KacperMadej, this looks like it should have been fixed by #43 as well. Missed this one when listing out the issues. I think you can close
